Python/ceval.c

```diff
@@ -394,6 +393,13 @@ const _Py_SpecialMethod _Py_SpecialMethods[] = {
     }
 };
 
+const size_t _Py_FunctionAttributeOffsets[] = {
```

Comment:
This one could be `static const` or can't it?

Reply:
The JIT needs to be able to find the symbol, so it can't be `static`